cernopendata/modules/fixtures/data/records/cms-primary-datasets-example.json

```diff
@@ -0,0 +1,336 @@
+[
```

Comment:
Instead of adding a fully new example, can you please enrich some existing file such as `cms-primary-datasets.json` if you have real container image data for them?